venues/ICLR.cc/2019/Conference/python/post-submissions-stage.py

```diff
@@ -54,6 +54,33 @@
 
     reviewers_group = client.get_group(iclr19.REVIEWERS_ID)
     areachairs_group = client.get_group(iclr19.AREA_CHAIRS_ID)
+    all_emails = reviewers_group.members + areachairs_group.members
     openreview.tools.replace_members_with_ids(client, reviewers_group)
     openreview.tools.replace_members_with_ids(client, areachairs_group)
     groups.update_homepage(client, '../webfield/homepagePostSubmission.js')
+
+    # User registration.
+
+    '''
+    This is a workaround to resolve the issue where not all reviewers/ACs
+    see the ICLR venue in their "your active venues" list.
+
+    Notice that we're falsifying the tauthor. This is so that the query
+    in home.js (copied below) will pick up the note as a reply,
+    but the note won't show up in the user's activity.
+
+    home.js, Line 67:
+    controller.get('invitations', { invitee: true, duedate: true, details: 'replytoNote,repliedNotes' })
+    '''
+    register_user_inv = client.post_invitation(iclr19.register_user_inv)
+    for email in all_emails:
+        register_note = client.post_note(openreview.Note.from_json({
+            'invitation': register_user_inv.id,
+            'forum': None,
+            'replyto': None,
+            'readers': register_user_inv.reply['readers']['values'],
+            'writers': register_user_inv.reply['writers']['values'],
+            'signatures': register_user_inv.reply['signatures']['values'],
+            'content': {'registered': 'yes'},
+            'tauthor': email
```

Comment:
I'm not sure if you can set the tauthor, it is an internal field

Reply:
I tested this before making this PR, and it works

Reply:
It seems as a superuser you can set the tauthor, I just checked it.